Scripts/Editor/Tests/anvil-unity-tests.asmdef

```diff
@@ -1,7 +1,11 @@
 {
     "name": "anvil-unity-tests",
     "rootNamespace": "Anvil.Unity.Tests",
-    "references": [],
+    "references": [
+        "anvil-unity-core-runtime",
+        "anvil-unity-core-editor",
+        "anvil-csharp-core"
+    ],
```

Comment:
Glad this works today. Might need to split into Editor and Runtime tests some day in the future.
Seems to be common in other packages
![image](https://user-images.githubusercontent.com/1426795/195736285-6fb0d9e9-7708-4ed2-8145-2faa4997cc63.png)

Reply:
Unity does support "Play mode" unit tests as well, which I assume is the distinction here - I only bothered setting up edit mode tests for now, but yeah it should be possible to do the same setup in the runtime folder, and do some fancy in-game testing too.